### ISSUE TITLE
Issue: Animation of subtask delete buttons makes task name weirdly move

### ISSUE THREAD

**Issue by esszed** [alainm23/planify#539]
**Describe the bug**
When subtask is too long to be shown whole, while the animation of delete button for the subtask is active the text slightly moves from left to right and once the animation is done the text ends up few pixels offset. 

**To Reproduce**
1. Create a new task
2. Create a new subtask (checklist) 
3. Make subtask text to be long enough that it can't fit 
4. Move cursor over the subtask
5. See the text moving with the animation

**Expected behavior**
The subtask should not be moving

**Screenshots**
![2020-09-28 11-26-59](https://user-images.githubusercontent.com/56234798/94423800-7d580580-0189-11eb-8ac0-7426670a51b2.gif)
![Screenshot_20200928_125304](https://user-images.githubusercontent.com/56234798/94423869-a082b500-0189-11eb-8900-08c520f9c3a1.png)

**Desktop:**
 - OS or DE: Manjaro KDE Plasma
 - Version 19.0.2
